Demo/Common/IntegrationExamples/Components/CardComponentExample.swift

```diff
@@ -0,0 +1,200 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import AdyenActions
+import AdyenCard
+import AdyenComponents
+import AdyenDropIn
+import PassKit
+import UIKit
+
+internal final class CardComponentExample: AdvancedFlowExampleProtocol {
```

Comment:
If this is `AdvancedFlowExampleProtocol` - this should be `AdvancedCardComponentExample`, right?

Reply:
The same was suggested by Eren :) I have renamed it. 47d133f7b00e66bf4ba0bc8650d603e28cc74c6d